### PR TITLE
Update s6 Stack and Mailspring SlackBuild

### PR DESCRIPTION
**Pull request by mumahendras3**
Update the entire s6 stack and Mailspring SlackBuild for their latest versions.

Thanks!

### CODE REVIEW

**Comment by aclemons** [comment]
@sbo-bot: build libraries/skalibs

**Comment by aclemons** [comment]
@sbo-bot: build network/mailspring

**Comment by aclemons** [comment]
@sbo-bot: build system/execline

**Comment by aclemons** [comment]
@sbo-bot: build system/s6-linux-init

**Comment by aclemons** [comment]
@sbo-bot: build system/s6-rc

**Comment by aclemons** [comment]
@sbo-bot: build system/s6

File: libraries/skalibs/skalibs.info [hunk 1]
```diff
@@ -1,8 +1,8 @@
 PRGNAM="skalibs"
-VERSION="2.13.1.1"
+VERSION="2.14.0.0"
 HOMEPAGE="https://skarnet.org/software/skalibs/"
-DOWNLOAD="https://skarnet.org/software/skalibs/skalibs-2.13.1.1.tar.gz"
-MD5SUM="2ede218c3583777cefcfc50a82683fbd"
+DOWNLOAD="https://skarnet.org/software/skalibs/skalibs-2.14.0.0.tar.gz"
+MD5SUM="be8d8069ec9162b41b95a0907ea8c2d5"
```
Comment (aclemons) [inline libraries/skalibs/skalibs.info:5]: @mumahendras3 Seems this md5sum is now:

`34e15729afc9428cf1814ffebb99543a  skalibs-2.14.0.0.tar.gz` Could you confirm and update please?

**Comment by willysr** [comment]
@mumahendras3 can you update MD5SUM for skalibs? that prevents this PR to build correctly

**Comment by mumahendras3** [comment]
> @mumahendras3 can you update MD5SUM for skalibs? that prevents this PR to build correctly

Done. Thanks for the heads-up!

**Comment by willysr** [comment]
@sbo-bot: build libraries/skalibs

**Comment by willysr** [comment]
@sbo-bot: build network/mailspring

**Comment by willysr** [comment]
@sbo-bot: build system/execline

**Comment by willysr** [comment]
@sbo-bot: build system/s6-linux-init

**Comment by willysr** [comment]
@sbo-bot: build system/s6-rc

**Comment by willysr** [comment]
@sbo-bot: build system/s6

**Comment by willysr** [comment]
merged

**Pull request closed without merging**